api/lib/searchOperators.js

```diff
@@ -8,6 +10,27 @@ const constantsObj = {
   DecSearchOperators: [43, 45, 42, 34, 39],
 };
 
+// Pairing the bindi characters with their non-bind counterparts
+const bindiCharsUni = {
```

Comment:
We can go two ways here.

1. We can either have an object that is not only limited to bindiChars (change name to altChars or something similar), and then we can use exact characters in it to make code readable.

2. However, if we are only going to limit it to bindpair, then we don't need to convert it into ascii codes using anvaad.js , we can just directly enter those ascii codes here and maybe add a single line comment to explain. That will remove the need for importing anvaad.js and multiple anvaadjs calls.

I prefer we go with `#2` but wouldn't mind going with `#1` for future compatibility as well.

Reply:
Went with the second suggestion, we can do the first one, if and when the need arises.